client/src/app/nav-bar/nav-bar.component.html

```diff
@@ -1,10 +1,12 @@
 <nav class="navbar navbar-marketing navbar-expand-lg bg-white navbar-light">
   <div class="container px-5">
-      <a class="navbar-brand text-dark" routerLink="/">Misty Mountain Financial</a>
+      <a class="navbar-brand text-dark" [routerLink]="['/']" routerLinkActive="active">Misty Mountain Financial</a>
```

Comment:
May Need Editing to <a class="navbar-brand text-dark" routerLink="/">Misty Mountain Financial</a>